pages/linux/light.md

```diff
@@ -0,0 +1,19 @@
+# ligth
+
+> CLI to control backlight.
+
+- Get the current backlight value in percent:
+
+`light`
+
+- Set the backlight value to percent:
+
+`light -S {{percent}}`
+
+- Reduce percentage to the current backlight value:
```

Comment:
I guess `Reduce given percentage from the current backlight value` makes more sense ?